README.md

```diff
@@ -65,14 +65,23 @@ We welcome contributions to amp.dev.
       ```
    1. Run the following command to use a faster YAML parser.
       ```sh
-      $ sudo apt install -y python-yaml
+      $ sudo apt install -y python-yaml libyaml-dev
       ```
 
 1. Install [Grow](http://grow.io), the static site generator used to build amp.dev. Do so by using `pip` instead of its installer. Using `pip` will enable importing from the `grow` package in Python later on.
 
    **Note**: Be sure to use the `pip` command associated with Python 3 as Grow 1 depends on Python 3.
 
+
+  **Mac**
+  ```sh
+    LDFLAGS="-L$(brew --prefix)/lib" CFLAGS="-I$(brew --prefix)/include" pipenv run pip3 install --global-option="--with-libyaml" --force pyyaml
```

Comment:
pipenv is never mentioned before

Reply:
Also when using pipenv `pip` commands should never be run with either `pipenv run` or `pipenv shell` but `pipenv install` should be used instead 🙂

Reply:
apologizes, that was copied straight from the [grow page](https://grow.io/libyaml/), and was left in mistakenly